ts/src/pro/htxpro.ts

```diff
@@ -0,0 +1,15 @@
+
+//  ---------------------------------------------------------------------------
+
+import huobi from './huobi.js';
+
+// ---------------------------------------------------------------------------
+
+export default class htxpro extends huobi {
```

Comment:
@ttodua why calling it `pro` ?

Reply:
in whole, I think I have to close this PR. we'd better to move into HTX fully and make `huobi` as alias instead.